addon/lib/document.js

```diff
@@ -127,6 +127,30 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => {
+      const nums = arr.filter((num) => !isNaN(num));
+      return nums.length ? Math.min(...nums) : null;
+    });
+    documentJexl.addTransform("max", (arr) => {
+      const nums = arr.filter((num) => !isNaN(num));
```

Comment:
Maybe extract this so it's not duplicated? Something like `const onlyNumbers = (nums) => nums.filter((num) => !isNaN(num));`

Reply:
Done!